### PR TITLE
Fix wording for /status

### PR DESCRIPTION
**Pull request by dns2utf8**

### CODE REVIEW

File: src/spaceapi_client.rs [hunk 1]
```diff
@@ -137,7 +137,7 @@ fn aggregate_status(status : Status) -> Result<String, String> {
     let pnp = match extract_sensors(sensors.people_now_present, "people_now_present") {
       Ok(o) => {
         o.into_iter().map(|e| {
-  format!("In {} are {} people.\n", e.location.unwrap_or_else(|| "unknown".into()), e.value)
+  format!("There are {n} people at {location}.\n", location=e.location.unwrap_or_else(|| "unknown".into()), n=e.value)
```
Comment (dbrgn) [inline src/spaceapi_client.rs]: I think you can also do `e.location.unwrap_or("unknown location".to_string())`.

**Comment by dbrgn** [comment]
Otherwise good!

**Comment by dns2utf8** [comment]
I had it like this first, but the `unwrap_or` is not implemented:

``` rust
error: no method named `unwrap_or` found for type `spaceapi::Optional<std::string::String>` in the current scope
```

**Comment by dbrgn** [comment]
Ah, that's our `optional` type... We need to get rid of it :) See https://github.com/coredump-ch/spaceapi-rs/issues/25

**Comment by dns2utf8** [comment]
I can change it if you like https://github.com/coredump-ch/spaceapi-rs/pull/27

**Comment by dbrgn** [comment]
I'll release spaceapi 0.3.0 soon, then you can use it directly.

**Pull request merged**